packages/components/src/panel/README.md

```diff
@@ -127,6 +127,20 @@ An icon to be shown next to the `PanelBody` title.
 -   Type: `String`
 -   Required: No
 
+###### iconOpen
+
+An icon to be shown as the toggle button within the `PanelBody` title when open.
+
+-   Type: `String`
+-   Required: No
+
+###### iconClosed
+
+An icon to be shown as the toggle button within the `PanelBody` title when closed.
+
+-   Type: `String`
+-   Required: No
```

Comment:
These props are not `String`s — let's update the description as follow:

```suggestion
An icon to be shown as the toggle button within the `PanelBody` title, in its collapsed state. See [the `@wordpress/icons` package](https://github.com/WordPress/gutenberg/blob/trunk/packages/icons/README.md) for a list of available icons.

-   Type: `React.ReactNode`
-   Required: No
-   Default: `chevronDown` icon from `@wordpress/icons`
```

Reply:
Done.